vulture_os/toolkit/api_parser/cortex_xdr/cortex_xdr.py

```diff
@@ -216,10 +222,21 @@ def format_log(self, log, kind):
     def execute(self):
         for kind in self.event_kinds.keys():
             logger.info(f"[{__parser__}]:execute: Getting {kind}", extra={'frontend': str(self.frontend)})
+            # Prevent requesting too newer alerts, sometimes case_id is empty and added later
+            # Delay of 2 minutes seems acceptable
+            to = timezone.now()-timedelta(minutes=2)
+            since = getattr(self, f"cortex_xdr_{kind}_timestamp") or timezone.now()-timedelta(hours=1)
```

Comment:
```suggestion
            since = getattr(self, f"cortex_xdr_{kind}_timestamp") or timezone.now()-timedelta(hours=1)
            to = min(timezone.now()-timedelta(minutes=2), since + timedelta(hours=24))
```